clustering_ranges_walker.hh

```diff
@@ -313,6 +323,9 @@ public:
 
     // Returns the upper bound of the last range in provided ranges set
     position_in_partition_view uppermost_bound() const {
```

Comment:
It seems that this function is only called from mutation_fragment_filter::uppermost_bound which is never used so maybe it's easier to just remove it. I will send a patch.